test/browser/browser.sh

```diff
@@ -37,6 +37,8 @@ if grep -q 'ID=.*fedora' /etc/os-release && [ "$PLAN" = "main" ]; then
     dnf install -y tcsh
     # required by TestJournal.testAbrt*
     dnf install -y abrt abrt-addon-ccpp reportd libreport-plugin-bugzilla libreport-fedora
+    # required by TestTeam
+    dnf install -y NetworkManager-team
```

Comment:
What about RHEL 9?

Reply:
The package `Recommends:` it, so it works fine. See passing testing-farm:centos-stream-9-x86_64:self . This is more of a "belt and suspenders" kind of thing.

Reply:
Ah, ok!